app/vmctl/vm_native.go

```diff
@@ -370,3 +371,15 @@ func buildMatchWithFilter(filter string, metricName string) (string, error) {
 	match := "{" + strings.Join(filters, " or ") + "}"
 	return match, nil
 }
+
+func removeDuplicateStr(arrIn []string) []string {
```

Comment:
Can we check for duplicates without creating a map?

Reply:
removed function and used map